modules/appeals_api/app/workers/appeals_api/decision_review_report_job.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+
+module AppealsApi
+  class DecisionReviewReportJob
+    include Sidekiq::Worker
+
+    def perform
```

Comment:
1. Let's take the `to` and `from` dates as default keyword arguments so that if we need to run this manually due to a delivery failure, we can do it by a `AppealsApi::DecisionReviewReportJob.perform(from: 3.days.ago.utc)` or similar.
1. If the schedule is only set to run MON-FRI, how are we going to consolidate all the submissions that may have happened over the weekend? `1.day.ago` isn't going to cover it.